strictdoc/export/html/generators/view_objects/document_screen_view_object.py

```diff
@@ -193,11 +194,11 @@ def render_document_version(self) -> Optional[str]:
         if self.document.config.version is None:
             return None
 
-        def resolver(variable_name):
+        def resolver(variable_name: str) -> str:
             if variable_name == "GIT_VERSION":
-                return self.git_client.get_commit_hash()
+                return self.git_client.get_commit_hash() or ""
```

Comment:
It turns out that `get_commit_hash` always returns str, so we could remove the `or`s here.